packages/@interactjs/utils/is.ts

```diff
@@ -25,8 +25,8 @@ const element = (thing: any): thing is Element => {
   // eslint-disable-next-line import/no-named-as-default-member
   const _window = win.getWindow(thing) || win.window
 
-  return /object|function/.test(typeof _window.Element)
-    ? thing instanceof _window.Element // DOM2
+  return /object|function/.test(typeof HTMLElement)
+    ? (thing instanceof HTMLElement || thing instanceof _window.Element)
     : thing.nodeType === 1 && typeof thing.nodeName === 'string'
 }
```

Comment:
This should be checking for `Element`, to also include `SVGElement`s. Something like this (I haven't tested it)

```suggestion
  return /object|function/.test(typeof _window.Element)
    ? thing instanceof _window.Element // DOM2
    : /object|function/.test(typeof Element)
      ? (thing instanceof Element || thing instanceof _window.Element)
      : thing.nodeType === 1 && typeof thing.nodeName === 'string'
}
```

Reply:
I tried your solution unfortunately it doesn't work with the bug I saw.

But I completely agree for the Element object, it makes sense :)

I did several tests, and I changed my commit.